src/components/RangeSlider/components/SingleThumb/SingleThumb.scss

```diff
@@ -29,6 +29,11 @@ $range-thumb-shadow-focus: 0 0 0 rem(1px) color('indigo');
   align-items: center;
   flex: 1 1 auto;
   height: $range-thumb-size;
+
+  input {
+    padding: spacing() 0;
+    background-color: transparent;
+  }
```

Comment:
The default browser cursor, in this case `pointer`, will not respect padding for the input so this will need an explicit `cursor: pointer;`